module/input/GameController/src/GameControllerData.hpp

```diff
@@ -105,7 +105,7 @@ namespace module::input::gamecontroller {
         GameType gameType;                       // type of the game (GAME_ROUNDROBIN, GAME_PLAYOFF, GAME_DROPIN)
         State state;                             // state of the game (STATE_READY, STATE_PLAYING, etc)
         bool firstHalf;                          // 1 = game in first half, 0 otherwise
-        TeamColour kickOffTeam;                  // the team number of the next team to kick off or DROPBALL
+        uint8_t kickOffTeam;                     // the team number of the next team to kick off or DROPBALL
```

Comment:
Is this something like 0 = us, 1 = them?

Reply:
Great question. Worth clarifying that here on the `.hpp`

Reply:
Based on the comment, `team number` would likely refer to our actual team number (14 or something for us)

Reply:
This is the team number, as assigned by the Robocup TC. For example, our team number for this Robocup is 12, I think.